diff --git a/dist/esm/index.js b/dist/esm/index.js
index d966ba81c2e5d17b395a3d0d1e0bca88537274ed..9c4e05375ef33bdf77359679227fa5fbccabbed8 100644
--- a/dist/esm/index.js
+++ b/dist/esm/index.js
@@ -15115,7 +15115,7 @@ function registerTagWhitelistIfPossible(tagName) {
     }
 }
 function sanitizeHTML(html, options) {
-    return purify.sanitize(html, __assign$1({ ADD_TAGS: whiteTagList, ADD_ATTR: ['rel', 'target', 'hreflang', 'type'], FORBID_TAGS: [
+    return purify.sanitize(html, __assign$1({ ADD_TAGS: Array.from(new Set([...whiteTagList,"iframe"])), ADD_ATTR: ['rel', 'target', 'hreflang', 'type'], FORBID_TAGS: [
             'input',
             'script',
             'textarea',
diff --git a/dist/esm/indexViewer.js b/dist/esm/indexViewer.js
index 01bb8afd62d4a7383699332693cd29641775a90e..dc3a4a424a8a153451764ecfa822818c6427ec6f 100644
--- a/dist/esm/indexViewer.js
+++ b/dist/esm/indexViewer.js
@@ -11539,7 +11539,7 @@ function registerTagWhitelistIfPossible(tagName) {
     }
 }
 function sanitizeHTML(html, options) {
-    return purify.sanitize(html, __assign$1({ ADD_TAGS: whiteTagList, ADD_ATTR: ['rel', 'target', 'hreflang', 'type'], FORBID_TAGS: [
+    return purify.sanitize(html, __assign$1({ ADD_TAGS: Array.from(new Set([...whiteTagList,"iframe"])), ADD_ATTR: ['rel', 'target', 'hreflang', 'type'], FORBID_TAGS: [
             'input',
             'script',
             'textarea',
diff --git a/dist/toastui-editor-viewer.js b/dist/toastui-editor-viewer.js
index 8dac9c7f0ead1a0c2366c297f6b71c84bebafef5..e808a3779bf4778c7235afe2a1b62783a248bebb 100644
--- a/dist/toastui-editor-viewer.js
+++ b/dist/toastui-editor-viewer.js
@@ -12839,7 +12839,7 @@ function htmlSanitizer_registerTagWhitelistIfPossible(tagName) {
     }
 }
 function sanitizeHTML(html, options) {
-    return purify_default().sanitize(html, tslib_es6_assign({ ADD_TAGS: whiteTagList, ADD_ATTR: ['rel', 'target', 'hreflang', 'type'], FORBID_TAGS: [
+    return purify_default().sanitize(html, tslib_es6_assign({ ADD_TAGS: Array.from(new Set([...whiteTagList,"iframe"])), ADD_ATTR: ['rel', 'target', 'hreflang', 'type'], FORBID_TAGS: [
             'input',
             'script',
             'textarea',
diff --git a/dist/toastui-editor.js b/dist/toastui-editor.js
index dd572b1daead0aa3742426b40d631f66629ce064..9189512d40d067917ab22c860ec72dacb6b32956 100644
--- a/dist/toastui-editor.js
+++ b/dist/toastui-editor.js
@@ -16370,7 +16370,7 @@ function registerTagWhitelistIfPossible(tagName) {
     }
 }
 function sanitizeHTML(html, options) {
-    return purify_default().sanitize(html, __assign({ ADD_TAGS: whiteTagList, ADD_ATTR: ['rel', 'target', 'hreflang', 'type'], FORBID_TAGS: [
+    return purify_default().sanitize(html, __assign({ ADD_TAGS: Array.from(new Set([...whiteTagList,"iframe"])), ADD_ATTR: ['rel', 'target', 'hreflang', 'type'], FORBID_TAGS: [
             'input',
             'script',
             'textarea',
